### DISCUSSION TITLE
Fallback to default locale when translation file is missing for a locale

### DISCUSSION THREAD

**Discussion by josephfarina** [vinissimus/next-translate#391] [Ideas]
Hey!

Is it possible to use the `defaultLocale` translation file when one for the current language doesn't exist?

**Comment by aralroca** [comment]
The problem I see is that then each page will have to load double kb in translations (current language + default language). Maybe we can make it configurable, disabled by default...

**Reply by jasonwilliams** [reply]
+1 on this. This is something other translations libraries provide so i was suprised to see it didn't fallback out the box. I understand its for performance reasons. Maybe there could be some opt-in like you described.

btw im not sure if I should be voting for this feature here or [here](https://github.com/vinissimus/next-translate/issues/463)
